src/test/ui/issues/issue-76191.stderr

```diff
@@ -0,0 +1,22 @@
+error[E0308]: mismatched types
+  --> $DIR/issue-76191.rs:10:9
+   |
+LL | const RANGE: RangeInclusive<i32> = 0..=255;
+   | ------------------------------------------- constant defined here
+...
+LL |     match n {
+   |           - this expression has type `i32`
+LL |         RANGE => {}
+   |         ^^^^^
+   |         |
+   |         expected `i32`, found struct `std::ops::RangeInclusive`
+   |         `RANGE` is interpreted as a constant, not a new binding
+   |         help: introduce a new binding instead: `other_range`
+   |
+   = note: expected type `i32`
+            found struct `std::ops::RangeInclusive<i32>`
+   = note: Constants only support matching the same type. If you are using range syntax (i.e. ..= or similar) in your constant you may want to move the range into the match block
```

Comment:
We don't use "proper" sentences, no leading uppercase letter and no period after the sentence.